makefile

```diff
@@ -0,0 +1,8 @@
+# broken linux makefile, committed for testing and posterity. do not use -- ali
+
+make:
```

Comment:
Since this is a broken makefile, I would either comment out everything or make it a markdown file instead so it doesn't break anything in case it gets accidentally run